backend/src/main/java/gov/cdc/usds/simplereport/config/InitialSetupProperties.java

```diff
@@ -0,0 +1,76 @@
+package gov.cdc.usds.simplereport.config;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConstructorBinding;
+
+import gov.cdc.usds.simplereport.db.model.DeviceType;
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.db.model.Provider;
+import gov.cdc.usds.simplereport.db.model.StreetAddress;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PersonName;
+
+@ConfigurationProperties(prefix = "simple-report-initialization")
+@ConstructorBinding
+public class InitialSetupProperties {
+
+	private Organization organization;
+	private ConfigProvider provider;
+	private List<? extends ConfigDeviceType> deviceTypes;
+	private List<String> configuredDeviceTypes;
+
+	public InitialSetupProperties(ConfigOrganization organization, ConfigProvider provider,
+			List<ConfigDeviceType> deviceTypes, List<String> configuredDeviceTypes) {
+		this.organization = organization;
+		this.provider = provider;
+		this.deviceTypes = deviceTypes;
+		this.configuredDeviceTypes = configuredDeviceTypes;
+	}
+
+	public List<String> getConfiguredDeviceTypeNames() {
+		return configuredDeviceTypes;
+	}
+
+	public Organization getOrganization() {
+		return organization;
+	}
+
+	public Provider getProvider() {
+		return provider.getRealProvider();
+	}
+
+	public List<? extends DeviceType> getDeviceTypes() {
+		return deviceTypes.stream().map(ConfigDeviceType::getReal).collect(Collectors.toList());
+	}
+
+	private static final class ConfigOrganization extends Organization {
+		@ConstructorBinding
+		public ConfigOrganization(String facilityName, String externalId, String cliaNumber) {
+			super(facilityName, externalId, cliaNumber);
+		}
+	}
+	public static final class ConfigProvider extends Provider {
```

Comment:
I could have just bypassed this by annotating `Provider` with `@ConstructorBindimg` but I was trying to avoid mucking with the upstream models. Probably a mistake, but in the long term this whole thing is probably better off axed anyway.